libs/sdk-flutter/lib/bridge_generated.dart

```diff
@@ -258,6 +258,11 @@ abstract class BreezSdkCore {
 
   FlutterRustBridgeTaskConstMeta get kRescanSwapsConstMeta;
 
+  /// See [BreezServices::redeem_swap]
+  Future<void> redeemSwap({required String swapAddress, dynamic hint});
```

Comment:
This function needs to be exposed through `BreezSDK`(_sdk-flutter/lib/sdk.dart_) to be accessible from the plugin.

Reply:
Good catch.

Is this the right way to do it? 67c89ee0132c554ddf7553d39f09f5d590aff1b4